src/ocean/text/convert/Float.d

```diff
@@ -94,6 +94,30 @@ private enum
 
 /******************************************************************************
 
+  Defines information needed to format Commas and decimals.
```

Comment:
I think this is more informative: "Defines information needed to format numbers with digit grouping and/or decimal marks."